cmd/openfga/openfga.go

```diff
@@ -210,6 +210,8 @@ func buildService(logger logger.Logger) (*service, error) {
 		return nil, fmt.Errorf("failed to initialize authenticator: %v", err)
 	}
 
+	logger.Info(fmt.Sprintf("using '%v' authentication", config.AuthMethod))
```

Comment:
If `config.AuthMethod == "none"` can we emit a warning instead of an info statement?